Source/Fuse.Common/Timer.uno

```diff
@@ -29,8 +29,8 @@ namespace Fuse
 
 		void Stop()
 		{
-			_running = false;
 			UpdateManager.RemoveAction(Update);
+			_running = false;
```

Comment:
No, `_running` was already being set to `false`, you just moved it below the `UpdateManager.RemoveAction(Update);`-line. That's just code-churn, and doesn't improve anything.